components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -258,10 +259,29 @@ public static boolean isValidCaptcha(String reCaptchaResponse) throws CaptchaExc
         try {
             try (InputStream in = entity.getContent()) {
                 JsonObject verificationResponse = new JsonParser().parse(IOUtils.toString(in)).getAsJsonObject();
-                if (verificationResponse == null || verificationResponse.get("success") == null ||
-                        !verificationResponse.get("success").getAsBoolean()) {
+                if (verificationResponse == null) {
                     throw new CaptchaClientException("reCaptcha verification failed. Please try again.");
                 }
+                boolean success = verificationResponse.get(CaptchaConstants.CAPTCHA_SUCCESS) != null
+                        && verificationResponse.get(CaptchaConstants.CAPTCHA_SUCCESS).getAsBoolean();
+                if (verificationResponse.get(CaptchaConstants.CAPTCHA_SCORE) != null) {
+                    double score =  verificationResponse.get(CaptchaConstants.CAPTCHA_SCORE).getAsDouble();
```

Comment:
Double spaces after = mark

Reply:
Use wso2 code formatter to format the code.